### PR TITLE
CIF-2211 - Update core WCM dependency to latest for CIF components

### PR DESCRIPTION
**Pull request by LSantha**
Updated version number to 2.17.2 and fixed test failures.

## Related Issue

CIF-2211

## Motivation and Context

JUnit.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [X] All unit tests pass on CircleCi.
- [X] I ran all tests locally and they pass.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -72,13 +72,13 @@ executors:
...
         <<: *docker_auth
-      - image: docker-adobe-cif-release.dr-uw2.adobeitc.com/circleci-aem-cloudready:5343-azul
+      - image: docker-adobe-cif-release.dr-uw2.adobeitc.com/circleci-aem-cloudready:5607-azul
         <<: *docker_auth
   test_executor_655:
     docker:
       - image: docker-adobe-cif-release.dr-uw2.adobeitc.com/circleci-qp:6.4.4-azul-jdk11
         <<: *docker_auth
-      - image: docker-adobe-cif-release.dr-uw2.adobeitc.com/circleci-aem:6.5.8-azul
+      - image: docker-adobe-cif-release.dr-uw2.adobeitc.com/circleci-aem:6.5.9-azul
```
Comment (Buuhuu) [inline .circleci/config.yml:81]: should we update the readme saying we support 6.5.9+ instead 6.5.7 now?

Comment (mhaack) [reply .circleci/config.yml:81]: +1

Comment (mhaack) [reply .circleci/config.yml:81]: Plus we should keep the `uber-jar` version in parent pom.xml in sync as well.

Comment (LSantha) [reply .circleci/config.yml:81]: In the readme 6.5.7 appears as minimum system requirement.  This PR doesn't have anything that would require AEM 6.5.9, I've updated the CI build only to make sure that we are compatible with the latest AEM releases.
The uber-jar version is still AEM 6.4.4 . 
So, should we enforce AEM 6.5.9 then? (meaning whoever wants to use the CIF components should upgrade to AEM 6.5.9)

Comment (herzog31) [reply .circleci/config.yml:81]: If we test with 6.5.9, we would never find out if we introduced anything that breaks with 6.5.7.

Comment (mhaack) [reply .circleci/config.yml:81]: +1 we should test, require and document only one version.

>This PR doesn't have anything that would require AEM 6.5.9

If that is the case we should revert the container version if not really needed.

**Comment by LSantha** [comment]
I've reverted the AEM image to 6.5.8 here and in the Venia PR (https://github.com/adobe/aem-cif-guides-venia/pull/165).
Both are passing.

**Review by Buuhuu — APPROVED** [review]

**Review by mhaack — APPROVED** [review]

**Pull request merged**